WcaOnRails/app/models/competition.rb

```diff
@@ -378,8 +378,11 @@ def warnings_for(user)
       # NOTE: this will show up on the edit schedule page, and stay even if the
       # schedule matches when saved. Should we add some logic to not show this
       # message on the edit schedule page?
-      unless has_any_round_per_event? && schedule_includes_rounds?
-        warnings[:schedule] = I18n.t('competitions.messages.schedule_must_match_rounds')
+      # Competitions after 2018-7-24 will have this check, date set to avoid errors at old competitions
+      if start_date.present? && start_date > Date.new(2018, 7, 30)
+        unless has_any_round_per_event? && schedule_includes_rounds?
+          warnings[:schedule] = I18n.t('competitions.messages.schedule_must_match_rounds')
```

Comment:
This only shows a warning box on the competition page in general, but doesn't block editing. I believe you intended to modify this part:

https://github.com/thewca/worldcubeassociation.org/blob/e974e9020e8c8a1e562c57695b96b312efb8eafa/WcaOnRails/app/models/competition.rb#L242-L249

Reply:
You're right. I undid the first change (as it seems to only display before the competition is confirmed) and set the modification to the lines you mentioned.